src/howitz/endpoints.py

```diff
@@ -35,6 +39,40 @@ class EventColor(StrEnum):
     DEFAULT = ""
 
 
+# Fixme add non-generic error handling as well
+@main.errorhandler(HTTPException)
+def handle_exception(e):
+    """Return JSON instead of HTML for HTTP errors."""
+    # start with the correct headers and status code from the error
+    response = e.get_response()
+    # replace the body with JSON
+    response.data = json.dumps({
+        "code": e.code,
+        "name": e.name,
+        "description": e.description,
+    })
+    response.content_type = "application/json"
+    return response
+
+
+@main.errorhandler(Exception)
+def handle_exception(e):
+    # pass through HTTP errors
+    if isinstance(e, HTTPException):
+        return e
+
+    # now you're handling non-HTTP exceptions only
+    alert_random_id = randint(1, 100000)
```

Comment:
Safer to use uuid4. The chance of getting the same number twice in a row is not zero.